module/skill/QuinticWalk/data/config/quinticwalk.yaml

```diff
@@ -0,0 +1,131 @@
+log_level: WARN
```

Comment:
This config file should be `QuinticWalk.yaml` right?
I suppose you're trying to avoid a collision with the old walk.
I would change the old walk rather than this one.

Reply:
Old is fine, I made the new kick lowercase but making the old one lowercase is probably better